app/models/message.rb

```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
```

Comment:
- [ ] [REQUIRED] You have done a great job creating the model, however, following the controller name, your model should be named `Greeting` instead of `Message`. Rails naming convention requires that the model name should match the controller name.  If your controller is named GreetingsController, the corresponding model should be named Greeting.

Reply:
well resolved! ✔️